src/main/webapp/app/exercises/programming/manage/update/programming-exercise-update.component.html

```diff
@@ -538,6 +538,31 @@ <h4 *ngIf="isImport" id="jhi-programming-exercise-heading-import" jhiTranslate="
                 </label>
             </div>
         </div>
+        <div class="form-group mb-0" *ngIf="programmingExercise.programmingLanguage === ProgrammingLanguage.JAVA">
```

Comment:
add `this.isBamboo = profileInfo.activeProfiles.includes('bamboo');` in `ngOnInit` of the component and then integrate `&& isBamboo` into the `ngIf`